### PR TITLE
Create pr Template 

### PR DESCRIPTION
**Pull request by Linkevin1997**
resolving issue 'Create Pull request template https://github.com/Cosmic-Binturongs/slothstagram/issues/26'

### CODE REVIEW

**Comment by Linkevin1997** [comment]
Create Pull request Template for github #26. Issue resolved.

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Related issue(s): 
+- issue goes here
+
+# Description of changes made: 
+- change 1
+- change 2
+
+# (optional) Questions or comments for review:
+- any additional questions or comments goes here.
```
Comment (Underwaterr) [inline docs/pull_request_template.md:9]: Instead of using top-level headers with `#` let's switch this to sub-headings with `###`.

**Review by Underwaterr — CHANGES_REQUESTED** [review]

File: docs/pull_request_template.md (hunk 1 above)
Comment (Linkevin1997) [reply docs/pull_request_template.md]: resolving issue '# let's switch this to sub-headings with ### https://github.com/Cosmic-Binturongs/slothstagram/issues/26'

**Review by Underwaterr — CHANGES_REQUESTED** [review]
You changed the first heading to be a subheading, do the same for the next two

**Review by Underwaterr — APPROVED** [review]

**Pull request merged**